### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by nllong**
small changes and some instructions.

### CODE REVIEW

File: test.rb [hunk 1]
```diff
@@ -117,13 +118,6 @@ def duplicate_measure(old_measure_dir, measure_dir, name, class_name, taxonomy_t
   #puts info
 end
 
-measures.each do |measure|
```
Comment (macumber) [inline test.rb:120]: @nllong I added this here to test the measure caching behavior.  This script is a test script, in a real rake task or whatever you would not call this twice

File: measure_manager.rb [hunk 2]
```diff
@@ -136,7 +136,7 @@ def get_measure(measure_dir, force_reload)
       @measure_info[measure_dir] = {}
     end
     
-    if result
+    if result && !@measures[measure_dir].nil?
```
Comment (macumber) [inline measure_manager.rb:139]: @nllong this is a bug but should probably just set result to false on line 129, was this the blow up with no measure.rb file?

**Comment by macumber** [comment]
Thanks @nlong, can you put the duplicate calls back in the test to test the caching behavior and make the minor change to the bug fix?  After that I will merge this in.  Final location for this code should be in the OpenStudio CLI.  You can either call this stuff from command line or use the CLI to launch the measure manager server

**Comment by nlong** [comment]
Wrong @nlong :)

**Comment by macumber** [comment]
Doh!  Sorry @nlong that was meant for @nllong !

**Comment by nllong** [comment]
I’m guessing my 2016 merge isn’t going to happen ;)

**Comment by macumber** [comment]
I have no idea what this even was, glad you were able to get by without it.  Hope you are doing well!

**Pull request closed without merging**